Appraisals

```diff
@@ -305,15 +305,15 @@ elsif Gem::Version.new('2.2.0') <= Gem::Version.new(RUBY_VERSION) \
     appraise 'rails5-postgres-redis' do
       gem 'rails', '~> 5.2.1'
       gem 'pg', '< 1.0', platform: :ruby
-      gem 'redis'
+      gem 'redis', '~> 4'
```

Comment:
Shaking up our depencies has revealed that we are not requiring the minimum version of `redis` required by [Rails 5.2](https://github.com/rails/rails/blob/v5.2.0/activesupport/lib/active_support/cache/redis_cache_store.rb#L8).
This allowed `bundler` to install `redis ~> 3` in CI, which is not compatible with the version of Rails under test.

Reply:
Moved to its own PR: https://github.com/DataDog/dd-trace-rb/pull/890